syntax = "proto3";

import "trade/vo/MaxAvailableAssetVo.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "TradeQueryMaxAvailableAssetResponseProto";
message TradeQueryMaxAvailableAssetResponse {
	MaxAvailableAssetVo data = 1;
}
